src/git-command-manager.ts

```diff
@@ -11,7 +11,8 @@ import {GitVersion} from './git-version'
 
 // Auth header not supported before 2.9
 // Wire protocol v2 not supported before 2.18
-export const MinimumGitVersion = new GitVersion('2.18')
+// sparse-checkout not [well-]supported before 2.28 (see https://github.com/actions/checkout/issues/1386)
+export const MinimumGitVersion = new GitVersion('2.28')
```

Comment:
Consider an even higher version in light of https://github.com/actions/checkout/issues/1708.

`2.34.1` exhibits the problem reported in #1708.
`2.43.0` works, but there's probably an earlier candidate that would also work.

(Need to go through the git release notes between version range [2.34 - 2.43] and find where they addressed compatibility issues associated with `repositoryformatversion=1`)